test/parse.js

```diff
@@ -4,67 +4,67 @@ import fn from '../';
 // https://github.com/sindresorhus/query-string/pull/48#issuecomment-198242935
 function tsame(t, actual, expected) {
 	Object.keys(expected).forEach(key => {
-		t.is(actual[key], expected[key]);
+		if (Array.isArray(expected[key])) {
+			tsame(t, actual[key], expected[key]);
+		} else {
+			t.is(actual[key], expected[key]);
+		}
 	});
 }
 
-test.beforeEach(t => {
-	t.context.same = tsame.bind(undefined, t);
-});
```

Comment:
Not being used anymore - I'm explicitly passing in `t` to the tsame function. See comment above (and commit message for 6d80fe9469) about binding the function to the the test instance causing behaviour I don't fully understand.